components/Container.tsx

```diff
@@ -8,7 +8,7 @@ import cn from 'classnames';
 import Footer from 'components/Footer';
 import MobileMenu from 'components/MobileMenu';
 
-function NavItem({ href, text }) {
+const NavItem = ({ href, text }: { href: string; text: string }) => {
```

Comment:
Any particular reason this was changed from `function`?

Reply:
No particular reason. I just like arrow functions. 😆